src/Client.ts

```diff
@@ -32,6 +33,14 @@ export class RavenApiClient {
     return (this.#event ??= new EventClient(this.options));
   }
 
+  public async send(request: RavenApi.event.send.Request): Promise<RavenApi.event.send.Response> {
+    return this.event.send(request);
```

Comment:
lets call the `event()` instead of referencing the private member directly. 
so should be `event().send(request)`